### PR TITLE
Configuration for extractor to put specific projects in specific Example

### LINKED ISSUE #90: Configuration for extractor to put specific projects in specific Examp
We need to restore javadocsummary example with Japan period to run only on jdk and special project with Japan language javadoc comments.

### PR DESCRIPTION
**Pull request by relentless-pursuit**
Resolves #90

### CODE REVIEW

**Comment by relentless-pursuit** [comment]
Does having too many branches creates issue in git changes? like I am constantly facing issue of merge conflicts, have to rebase and other issues. Any clue?

**Comment by romani** [comment]
If you change the same line, yes , it is conflict.

File: extractor/src/main/resources/projects-for-example.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+SummaryJavadocCheck:
+  Example3:
```
Comment (romani) [inline extractor/src/main/resources/projects-for-example.yml:2]: SummaryJavadoc

File: extractor/src/main/resources/projects-for-example.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    projects:
+      - WxJava
```
Comment (romani) [inline extractor/src/main/resources/projects-for-example.yml:4]: Please add jdk to project list

File: extractor/src/main/java/com/example/extractor/YamlParserAndProjectHandler.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            }
+        } else {
+            fileContents.addAll(Files.readAllLines(Paths.get(DEFAULT_PROJECTS_FILE_PATH)));
+        }
+
+        try {
+            Files.write(projectsFilePath, fileContents);
+        } catch (IOException e) {
+            System.err.println("Error writing file: " + e.getMessage());
+            throw e;
```
Comment (romani) [inline extractor/src/main/java/com/example/extractor/YamlParserAndProjectHandler.java:66]: Do wrap to Illegal state exception with such message that you print and rethrow exception .
No println

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: extractor/src/main/resources/projects-for-example.yml (hunk 1 above)
Comment (relentless-pursuit) [reply extractor/src/main/resources/projects-for-example.yml:2]: I am waiting for my PR in the main repo to be merged. Once that is done, we can refactor.

File: extractor/src/main/java/com/example/extractor/YamlParserAndProjectHandler.java (hunk 3 above)
Comment (relentless-pursuit) [reply extractor/src/main/java/com/example/extractor/YamlParserAndProjectHandler.java]: done.

File: extractor/src/main/resources/projects-for-example.yml (hunk 2 above)
Comment (relentless-pursuit) [reply extractor/src/main/resources/projects-for-example.yml:4]: done.

File: extractor/src/main/resources/projects-for-example.yml (hunk 1 above)
Comment (romani) [reply extractor/src/main/resources/projects-for-example.yml:2]: @relentless-pursuit , that PR is merged. Please do update

Comment (relentless-pursuit) [reply extractor/src/main/resources/projects-for-example.yml:2]: > @relentless-pursuit , that PR is merged. Please do update

rebased.

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**